src/pint/utils.py

```diff
@@ -2762,7 +2980,9 @@ def sherman_morrison_dot(Ndiag, v, w, x, y):
     return result, logdet_C
 
 
-def woodbury_dot(Ndiag, U, Phidiag, x, y):
+def woodbury_dot(
+    Ndiag: np.ndarray, U: np.ndarray, Phidiag: float, x: np.ndarray, y: np.ndarray
```

Comment:
Phidiag is also an array.

Reply:
ok, thanks.  I had been following the docstring, but now both are fixed.